url-sampler/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
+  id("com.github.johnrengelman.shadow")
+
+  id("otel.java-conventions")
+  id("otel.publish-conventions")
+}
+
+description = "Sampler which makes its decision based on http.url semantic attribute"
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  implementation("io.opentelemetry:opentelemetry-sdk")
+  implementation("io.opentelemetry:opentelemetry-semconv")
+  implementation("org.slf4j:slf4j-simple")
+}
+
+tasks {
+  shadowJar {
```

Comment:
We don't need to shadow at least at this layer

Reply:
Why not? If I want to grab this sampler and add it as an extension to my agent, then I need a full shadowed jar, right?

Reply:
Well we definitely don't want to shade in the SDK as the agent distro will be bringing it in. This sampler doesn't seem to have any dependencies that we want to shade.

Anyways in your issue you also mention "At this moment I don't propose nor plan any auto-configuration support for this sampler." :P Let's just implement this as we would a normal Java library for now.